app/middleware/stripe-setup/check-company-number-not-submitted.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+// Local dependencies
+const { ConnectorClient } = require('../../services/clients/connector_client')
+const connector = new ConnectorClient(process.env.CONNECTOR_URL)
+const { renderErrorView } = require('../../utils/response')
+const paths = require('../../paths')
+
+module.exports = function checkVatNumberCompanyNumberNotSubmitted (req, res, next) {
+  if (!req.account) {
+    renderErrorView(req, res, 'Internal server error')
+    return
+  }
+
+  connector.getStripeAccountSetup(req.account.gateway_account_id, req.correlationId).then(stripeSetupResponse => {
+    if (stripeSetupResponse.companyNumber) {
+      req.flash('genericError', 'You’ve already provided your company registration number.<br />Contact GOV.UK Pay support if you need to update them.')
```

Comment:
should be "...if you need to update it."

Reply:
👍 updated